build.ps1

```diff
@@ -56,7 +56,7 @@ try {
 
     # Test
     Write-Message "Executing unit tests"
-    Get-DotNetProjectDirectory -RootPath $PSScriptRoot\test | Invoke-Test
+    Get-DotNetProjectDirectory -RootPath $PSScriptRoot\test | Where-Object { $_ -inotlike "*/Integration.*" } | Invoke-Test

```

Comment:
Is there a reason we can't run the integration tests as part of the build? I believe if you're using the in-memory httpclient test behaviour that it should work?

Reply:
We totally run the tests. What this does is exclude the _sites_ from being run like `dotnet test Integration.Net6` - it just creates log noise because there are no tests in the site assemblies. The actual tests that run are included and are in the A.E.DI.Integration.Test project, which isn't excluded by this.